SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -2465,13 +2479,11 @@ \subsection{Alignment}
 
 \small
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
-InputPixelType  m_ForegroundValue;
-OutputPixelType m_BackgroundValue;
+int                         m_ExtractionMode{ Self::LargestRegion };
+InputMeshPointType          m_ClosestPoint{};
```

Comment:
Good catch :Eyes:. I dropped the rest of the members since they had the same type as the ones on the next block, so it was a bad choice on my side. Changed for one that hopefully fits better in 841aa20.